Update db test secrets to allow passing of prodsec scanning.

tonyxrmdavidson opened the pull request:
This commit will allow the test credentials for the postgress and mysql databases to pass prodsec scanning.

It should stop emails being received by the team informing us we have a potential data leak because of test credentials.

Instructions on how should be done are found [here](https://source.redhat.com/departments/it/it-information-security/wiki/pattern_distribution_server#handling-false-positives).

## Description
Added '# notsecret tag to database-password and database-user lines in both postgres-db.yaml and mysql-db.yaml

Added a .gitleaks.toml file to allow prodsec to scan all previous commits.

## How Has This Been Tested?
It has been tested locally using gitleaks cli.

tarilabs reviewed (approved):
version bump and tests are passing 👍

tarilabs commented:
@dependabot squash and merge

tonyxrmdavidson committed:
Update db test secrets
This commit will allow the test credentials for the postgress and mysql databases to pass prodsec scanning

Signed-off-by: tonyxrmdavidson <tonyxrmdavidson@yahoo.co.uk>

tarilabs reviewed (commented):
/lgtm

thanks Tony

per my previous https://github.com/opendatahub-io/model-registry/pull/76#pullrequestreview-2147397232
if the samples manifests are taken from the ODH/MRO repo programmatically, one reason more to just need to maintain the `.gitleaks.toml` is the only source-of-truth place, my2c
